src/base/utils/misc.h

```diff
@@ -32,8 +32,13 @@
 #define UTILS_MISC_H
 
 #include <ctime>
+#include <memory>
```

Comment:
Are you sure this is needed?

Reply:
for `std::unique_ptr`, I'll move this into `#ifdef Q_OS_WIN` section.